sources/Notifications/Notifications.csproj

```diff
@@ -5,17 +5,17 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="AngleSharp" Version="1.0.0-alpha-844"/>
-        <PackageReference Include="Grpc.AspNetCore" Version="2.38.0"/>
-        <PackageReference Include="Sentry.Extensions.Logging" Version="3.8.3"/>
+        <PackageReference Include="AngleSharp" Version="1.0.0-alpha-844" />
+        <PackageReference Include="Grpc.AspNetCore" Version="2.38.0" />
+        <PackageReference Include="Sentry.Extensions.Logging" Version="3.8.3" />
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Database\Database.csproj"/>
+        <ProjectReference Include="..\Database\Database.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-        <Protobuf Include="..\..\assets\protos\*.proto" GrpcServices="Server"/>
+        <Protobuf Include="..\..\assets\protos\*.proto" GrpcServices="Server" />
```

Comment:
I was wondering, do we need to include *all* the protos when this service reads a single one? We might be bloating the bundle with unnecessary code.